test/libsolidity/syntaxTests/pragma/valid_range_operator_two_ranges.sol

```diff
@@ -0,0 +1,2 @@
+pragma solidity <0.8.15 || 0.8.17 - 0.8.20;
```

Comment:
We should also have a case where the prefix is next to `||` to make sure it is not handled differently:
```solidity
pragma solidity 0.8.15 || <0.8.17 - 0.8.20;
```